lib/conversion.ts

```diff
@@ -19,29 +19,24 @@ export function conversion(params: InsightsSearchConversionEvent) {
     );
   } else if (!params) {
     throw new Error(
-      "No parameters were sent to conversion event, please provide an objectID"
+      "No parameters were sent to conversion event, please provide a queryID and objectIDs"
     );
-  } else if (!params.objectID) {
+  } else if (!params.queryID) {
     throw new Error(
-      "No objectID was sent to conversion event, please provide an objectID"
+      "No parameters were sent to conversion event, please provide a queryID"
+    );
+  } else if (!params.objectIDs) {
+    throw new Error(
+      "No objectIDs was sent to conversion event, please provide objectIDs"
     );
   }
 
   // Get associated queryID
-  const queryID = params.queryID
-    ? params.queryID
-    : this.storageManager.getConversionObjectID(params.objectID);
```

Comment:
We no longer want to silently infer queryID for conversion, we'll provide later this as helper.
This could have been done in a later PR but I didn't want spend time adapting storageManager to multiple objectIDs